docs/source/setup/gee.rst

```diff
@@ -1,174 +1,178 @@
-Use Google Earth Engine with SEPAL
+Use Google Earth Engine (GEE) with SEPAL
 ==================================
 
 GEE and SEPAL
 -------------
 
-SEPAL is a cloud-computing platform that gives you access to geospatial data resources (either via Google Earth Engine data catalogue or user-provided data) and powerful computing resources to process these data to produce useful information. SEPAL is free to use and is open-source. 
+SEPAL is a free, open-source cloud-computing platform that gives you access to geospatial data resources (either via the GEE data catalogue or user-provided data) and powerful computing resources to process these data to produce useful information.
 
-SEPAL currently 'resides' in the Amazon AWS ecosystem and makes use of AWS cloud computers, or instances, that can be used to process data using pre-programmed SEPAL applications or using your own code (in R, Python or in the terminal). SEPAL is also closely linked to Google Earth Engine (GEE), a Google powered Earth-observation cloud-computing platform. SEPAL can access data stored in the GEE data catalog, use GEE for processing large datasets, and read and write data to GEE assets. A large part of SEPAL's existing code is based on the JavaScript GEE API.  
+SEPAL currently "resides" in the Amazon Web Services (AWS) ecosystem and makes use of AWS cloud computers, or instances, that can be used to process data using pre-programmed SEPAL applications or your own code (in R, Python or in the terminal). SEPAL is also closely linked to GEE, a Google-powered Earth-observation cloud-computing platform. SEPAL can access data stored in the GEE data catalog, use GEE for processing large datasets, and read and write data to GEE assets. A large part of SEPAL's existing code is based on the JavaScript GEE Application Programming Interface (API).  
 
-All the recipes of SEPAL are based on GEE and actually running scripts written by SEPAL team members to enable access to state-of-the-art processing algorithms without having to write code. In the SEPAL applications list, the applications tagged with the Google "G" logo are also running with the Python GEE API and don't require you to use big instances to run complex operations. 
+All SEPAL recipes are based on GEE and run scripts written by SEPAL team members to enable access to state-of-the-art processing algorithms without having to write code. In the SEPAL applications list, the applications tagged with the Google logo ("**G**") are also running with the Python GEE API and don't require you to use big instances to run complex operations. 
```

Comment:
```suggestion
All SEPAL recipes are based on GEE and run scripts written by SEPAL team members to enable access to state-of-the-art processing algorithms without having to write code. In the SEPAL applications list, the applications tagged with the Google logo (:icon:`fab fa-google`) are also running with the Python GEE API and don't require you to use big instances to run complex operations. 
```

since I wrote this page, I implemented an extention to use inline [font-awesome](https://fontawesome.com/v5/icons/google?s=brands) icons. let's use it here !